hnn_core/network.py

```diff
@@ -557,8 +569,16 @@ def add_poisson_drive(self, name, *, tstart=0, tstop=None, rate_constant,
         space_constant : float
             Describes lateral dispersion (from column origin) of synaptic
             weigths and delays within the simulated column
-        seedcore : int
+        probability : dict or float (default: 1.0)
+            Probability of connection between any src-target pair.
+            Use dict to create delay->cell mapping. If float, applies to all
```

Comment:
what is delay->cell mapping?

Reply:
Oops, typo. This was copied from the delay parameter for the drives API, which has identical data structures for input